src/charm.py

```diff
@@ -26,10 +23,6 @@
 logger = logging.getLogger(__name__)
 
 
-@trace_charm(
```

Comment:
by doing this you're effectively disabling charm-tracing. is that intentional?

Reply:
Yes, I removed the charm tracing integration to align this charm with the Loki worker, because the current integration was broken (due to part of the tracing being integrated in the **worker** shared object now).

Reply:
Now re-added!